pkgs/development/libraries/glibc/default.nix

```diff
@@ -36,6 +36,7 @@ in
       if [ -f ${stdenv.cc.cc}/lib/libgcc_s.so.1 ]; then
           mkdir -p $out/lib
           cp ${stdenv.cc.cc}/lib/libgcc_s.so.1 $out/lib/libgcc_s.so.1
+          patchelf --set-rpath $out/lib --force-rpath $out/lib/libgcc_s.so.1
```

Comment:
I don't see why ``--force-rpath`` is used. (I see now there's a nuked ref in RPATH unless we patchelf the lib.)